### PR TITLE
Create validatePR.yml

### PR DESCRIPTION
**Pull request by irvinesunday**
Adds new YAML file for build and test runs validations

### CODE REVIEW

**Review by andrueastman — DISMISSED** [review]

File: .github/workflows/validatePR.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+name: Validate PR
+
+on:  
+  pull_request:
```
Comment (thewahome) [inline .github/workflows/validatePR.yml:4]: Would you want to run it only when a PR is generated or even when someone makes subsequent pushes?

**Review by andrueastman — APPROVED** [review]

File: .github/workflows/validatePR.yml (hunk 1 above)
Comment (irvinesunday) [reply .github/workflows/validatePR.yml:4]: It still runs even on subsequent pushes against an opened PR. Ideally the `dev` branch should be closed off from direct pushes.

Comment (andrueastman) [reply .github/workflows/validatePR.yml:4]: https://docs.github.com/en/actions/reference/events-that-trigger-workflows#pull_request

Comment (thewahome) [reply .github/workflows/validatePR.yml:4]: Nice! I did not realize subsequent pushes were covered... 

> It still runs even on subsequent pushes against an opened PR. Ideally the `dev` branch should be closed off from direct pushes.

**Pull request merged**